WordPress/src/main/java/org/wordpress/android/util/config/FilterDiscoverFeatureConfig.kt

```diff
@@ -0,0 +1,20 @@
+package org.wordpress.android.util.config
+
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.annotation.FeatureInDevelopment
+import javax.inject.Inject
+
+/**
+ * Configuration of the filter for discover or topic feature.
+ */
+@FeatureInDevelopment
+class FilterDiscoverFeatureConfig
+@Inject constructor(appConfig: AppConfig) : FeatureConfig(
+        appConfig,
+        BuildConfig.ENABLE_FILTER_FOR_DISCOVER,
+        FILTER_DISCOVER
```

Comment:
While feature is in development, we don't need this third param `FILTER_DISCOVER`. 

```
* @param remoteField is the key of the feature flag in the remote config file, only set this field when there is a
* remote flag available
```

Let's just have below in the config file: 

```
@FeatureInDevelopment
class FilterDiscoverFeatureConfig
@Inject constructor(appConfig: AppConfig) : FeatureConfig(
        appConfig,
        BuildConfig.ENABLE_FILTER_FOR_DISCOVER
) 
```